modules/virt-jumbo-frames-vm-pod-nw.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc              
+
+:_content-type: CONCEPT                                            
+[id="virt-jumbo-frames-vm-pod-nw_{context}"]                                   
+= About jumbo frames support          
+
+When using the OVN-Kubernetes CNI plug-in, you can send unfragmented jumbo frame packets between two virtual machines (VMs) that are connected on the default pod network. Jumbo frames have a maximum transmission unit (MTU) value greater than 1500 bytes.
+
+The VM can get the MTU value from the pod NIC in two ways:
```

Comment:
I would like to emphasize that this should happen automatically and also to tie it into [Changing the MTU for the cluster network](https://57936--docspreview.netlify.app/openshift-enterprise/latest/networking/changing-cluster-network-mtu.html)

```suggestion
The VM will automatically pick up the MTU size configured on the cluster network using one of the two following methods:
```